FastBank.ConsoleApplication/MenuOptions.cs

```diff
@@ -205,9 +203,8 @@ static public void OpenCustomerMenu()
                 return;
             }
 
-            var bankAccountService = new BankAccountService();
             var customerBankAccount = bankAccountService.GetBankAccount((Customer)ActiveUser);
-            IMessageService MessageService = new MessageService();
+            IMessageService MessageService = new MessageService(bankAccountService);
```

Comment:
Why you are passing as parameter bankAccountService? If you do it, MessageService will have dependency and messageservice with be tightly coupled with bankaccountservice